src/components/tables/FieldSelection/FieldFilter.tsx

```diff
@@ -0,0 +1,54 @@
+import { TextField } from '@mui/material';
+import { debounce } from 'lodash';
+import { ChangeEvent, useRef } from 'react';
+import { useIntl } from 'react-intl';
+import {
+    useBinding_searchQuery,
+    useBinding_setSearchQuery,
+} from 'stores/Binding/hooks';
+import { TablePrefixes } from 'stores/Tables/hooks';
+
+interface Props {
+    disabled: boolean;
+}
+
+export default function FieldFilter({ disabled }: Props) {
+    const intl = useIntl();
+
+    const searchQuery = useBinding_searchQuery();
+    const setSearchQuery = useBinding_setSearchQuery();
+
+    const isFiltering = useRef(Boolean(searchQuery));
```

Comment:
Did not notice this before... why is this being stored? Seems like we are not doing anything with it.